indico/MaKaC/webinterface/common/timezones.py

```diff
@@ -67,6 +67,6 @@ def getSelectItemsHTML( self, selTitle="" ):
             if title==selTitle:
                 selected=" selected"
             l.append("""<option value=%s%s>%s</option>"""%(quoteattr(title),
-                                        selected, escape(title)))
+                                        selected, _(escape(title))))
```

Comment:
You can only translate static strings unless you ensure that the dynamic strings passed to `_()` are in the translation dict. However, this method is going away and the new string for it will be properly translated.